SDK/MobileCenterAnalytics/VSMC.Analytics.Windows.Shared/Analytics.cs

```diff
@@ -114,6 +129,17 @@ private void InstanceTrackEvent(string name, IDictionary<string, string> propert
             {
                 return;
             }
+            if (string.IsNullOrEmpty(name))
+            {
+                MobileCenterLog.Error(LogTag, "Name cannot be null or empty.");
+                return;
+            }
+            if (name.Length > MaxEventNameLength)
+            {
+                MobileCenterLog.Error(LogTag, "Name cannot be longer then " + MaxEventNameLength + " characters.");
```

Comment:
Messages should be synced with Android.
See https://github.com/Microsoft/mobile-center-sdk-android/pull/391#pullrequestreview-30926545